lib/src/pages/login.dart

```diff
@@ -0,0 +1,60 @@
+import "package:firebase_ui_oauth_google/firebase_ui_oauth_google.dart";
+import "package:flutter/material.dart";
+import "package:firebase_auth/firebase_auth.dart";
+import "package:google_sign_in/google_sign_in.dart";
+import "package:fluttertoast/fluttertoast.dart";
+import "package:study_buddies/data.dart";
+import "package:study_buddies/services.dart";
+
+
+class LoginPage extends StatelessWidget {
+  final GlobalKey<ScaffoldMessengerState> scaffoldMessengerKey = GlobalKey<ScaffoldMessengerState>();
```

Comment:
Not important, but for reference, this needs to be used in the widget tree somewhere to actually link it to the UI. In [this example](https://api.flutter.dev/flutter/material/ScaffoldMessenger/of.html), they use it in [`MaterialApp.scaffoldMessengerKey](https://api.flutter.dev/flutter/material/MaterialApp/scaffoldMessengerKey.html)